conf/config.json

```diff
@@ -90,6 +81,15 @@
             }
         },
         "lifecycle": {
+            "auth": {
+                "type": "service",
+                "account": "service-lifecycle",
```

Comment:
oh ok, as multiple such account can coexist with different canonical IDs? I re-used the terminology we used for the "auth" config objects.

Reply:
Maybe in case we re-generate a new service account because one is compromised, there could be a time where two of them co-exist before we remove the older one?

Reply:
In case a key is compromised Orbit can regenerate the key without changing the account canonical id.

I think it's safe to assume you'll have at most one account of type `service-lifecycle`